packages/services/api/src/modules/schema/providers/schema-publisher.ts

```diff
@@ -372,8 +372,10 @@ export class SchemaPublisher {
         }
       }
 
+      const hasSchemaChanges = !!checkResult?.state?.schemaChanges;
+
       schemaCheck = await this.storage.createSchemaCheck({
-        schemaSDL: sdl,
+        schemaSDL: hasSchemaChanges ? sdl : '',
```

Comment:
We want to insert `null` instead.

Reply:
just to double check. 

When I publish some version now for "single" project - I have empty "base_schema" field all the time in "schema_versions", and the field that is populated is "composite_sdl". should i use composite_sdl from "schema_versions" as a "fallback" or

Reply:
done